src/components/Fieldset/FieldsetLegend.tsx

```diff
@@ -0,0 +1,67 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import "lbh-frontend/lbh/components/lbh-fieldset/_fieldset.scss";
+
+/**
+ * The proptypes for {@link FieldsetLegend}.
+ */
+export interface FieldsetLegendProps {
+  id?: string;
+  className?: string;
+  isPageHeading?: boolean;
+  children: React.ReactNode;
+}
+
+/**
+ * A component to set a legend for a {@link Fieldset} by passing
+ * a {@link FieldsetProps.legend} prop.
```

Comment:
```suggestion
 * in via the {@link FieldsetProps.legend} prop.
```

I know I wrote this, but it reads weirdly.